static/app/components/profiling/flamegraph/flamegraphLayout.tsx

```diff
@@ -122,7 +123,7 @@ export function FlamegraphLayout(props: FlamegraphLayoutProps) {
             height={timelines.transaction_spans ? flamegraphTheme.SIZES.SPANS_HEIGHT : 20}
           >
             <CollapsibleTimeline
-              title="Transaction Spans"
+              title={t('Spans')}
```

Comment:
Should we just call this `Transaction` instead as that's what we call it in the performance product?